.codecov.yml

```diff
@@ -0,0 +1,18 @@
+coverage:
+  range: 80..100
+  round: down
+  precision: 2
+
+  status:
+    project:                   # measuring the overall project coverage
+      default:                 # context, you can create multiple ones with custom titles
+        enabled: yes           # must be yes|true to enable this status
+        target: 80%            # specify the target coverage for each commit status
```

Comment:
auto looks like a better option :)

Reply:
We used auto option in bowling project and it's not working good, but we can try do it in this project